docs/ImportantPythonFunctions.md

```diff
@@ -0,0 +1,180 @@
+Runnable IPython notebooks:
+- [load_model.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/load_model.ipynb)
+
+## Saving an ONNX Model
+```python
+import onnx
+
+onnx_model = ... # Your model in memory
+
+# Save the ONNX model
+with open('path/to/the/model', 'wb') as f:
+    f.write(onnx_model.SerializeToString())
+```
+Runnable IPython notebooks:
+- [save_model.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/save_model.ipynb)
+
+## Manipulating TensorProto and Numpy Array
+```python
+import numpy
+import onnx
+import os
+from onnx import numpy_helper
+
+# Preprocessing: create a Numpy array
+numpy_array = numpy.array([[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]], dtype=float)
+print('Original Numpy array:\n{}\n'.format(numpy_array))
+
+# Convert the Numpy array to a TensorProto
+tensor = numpy_helper.from_array(numpy_array)
+print('TensorProto:\n{}'.format(tensor))
+
+# Convert the TensorProto to a Numpy array
+new_array = numpy_helper.to_array(tensor)
+print('After round trip, Numpy array:\n{}\n'.format(numpy_array))
+
+# Save the TensorProto
+with open(os.path.join(os.path.dirname('tensor.pb'), 'wb') as f:
+    f.write(tensor.SerializeToString())
+
+# Load a TensorProto
+new_tensor = onnx.TensorProto()
+with open(os.path.join(os.path.dirname('tensor.pb'), 'rb') as f:
+    new_tensor.ParseFromString(f.read())
+print('After saving and loading, new TensorProto:\n{}'.format(new_tensor))
+```
+Runnable IPython notebooks:
+- [np_array_tensorproto.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/np_array_tensorproto.ipynb)
+
+## Creating an ONNX Model Using Helper Functions
+```python
+import onnx
+from onnx import helper
+from onnx import AttributeProto, TensorProto, GraphProto
+
+
+# The protobuf definition can be found here:
+# https://github.com/onnx/onnx/blob/master/onnx/onnx.proto
+
+
+# Create one input (ValueInfoProto)
+X = helper.make_tensor_value_info('X', TensorProto.FLOAT, [1, 2])
+
+# Create one output (ValueInfoProto)
+Y = helper.make_tensor_value_info('X', TensorProto.FLOAT, [1, 2])
+
+# Create a node (NodeProto)
+node_def = helper.make_node(
+    'Pad', # node name
+    ['X'], # inputs
+    ['Y'], # outputs
+    mode='constant', # Attributes
+    value=1.5,
+    pads=[0, 1, 0, 1],
+)
+
+# Create the graph (GraphProto)
+graph_def = helper.make_graph(
+    [node_def],
+    "test-model",
+    [X],
+    [Y],
+)
+
+# Create the model (ModelProto)
+model_def = helper.make_model(graph_def, producer_name='onnx-example')
+
+print('The model is:\n{}'.format(model_def))
+onnx.checker.check_model(model_def)
+print('The model is checked!')
+```
+Runnable IPython notebooks:
+- [make_model.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/make_model.ipynb)
+- [Protobufs.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/Protobufs.ipynb)
+
+## Checking an ONNX Model
+```python
+import onnx
+import os
+
+# Preprocessing: load the ONNX model
+model_path = os.path.join(os.path.dirname('model.onnx')
```

Comment:
unfinished line? is this code runnable? :)

Reply:
Yep, forgot to delete something.